components/style/properties/longhand/inherited_svg.mako.rs

```diff
@@ -165,74 +171,83 @@ ${helpers.predefined_type("marker-end", "UrlOrNone", "Either::Second(None_)",
     #[derive(Clone, Copy, Debug, PartialEq, ToComputedValue)]
     pub struct SpecifiedValue(pub u8);
 
+    impl SpecifiedValue {
+        fn normal() -> Self {
+            SpecifiedValue(0)
+        }
+    }
+
     pub mod computed_value {
         pub use super::SpecifiedValue as T;
     }
 
     pub fn get_initial_value() -> SpecifiedValue {
-      SpecifiedValue(NORMAL)
+        SpecifiedValue::normal()
     }
 
     impl SpecifiedValue {
-        pub fn bits_at(&self, pos: u8) -> u8 {
-            (self.0 >> pos * SHIFT) & MASK
+        fn order_at(&self, pos: u8) -> PaintOrder {
+            // Safe because PaintOrder covers all possible patterns.
+            unsafe { ::std::mem::transmute((self.0 >> pos * SHIFT) & MASK) }
```

Comment:
Why not just store a `PaintOrder` in `SpecifiedValue`?

Reply:
(There are three packed values, so it's not clear to me how could I do that)